performance/cinterop/src/nativeInterop/cinterop/types.def

```diff
@@ -0,0 +1,62 @@
+package = org.jetbrains.typesBenchmarks
+
+---
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <stdarg.h>
+
+int charFrequency(const char* string, char charToFind) {
+    int frequency = 0;
+    for(int i = 0; string[i] != NULL; ++i) {
+       if(charToFind == string[i])
+           ++frequency;
+    }
+    return frequency;
+}
+
+const char* findSuitableString(const char* stringsArray[], int arraySize, const char* stringToFind) {
+    for(int i = 0; i < arraySize; ++i) {
+        if (strcmp(stringsArray[i], stringToFind) == 0) {
+            return stringToFind;
```

Comment:
When called from Kotlin, `stringToFind` lifetime ends after return, so Kotlin code gets dangling pointer in this case.